### ISSUE TITLE
Failed to load grammar: `python-traceback.cson`

### ISSUE THREAD

**Issue by madiedgar** [legesher/language-legesher-python#2]
# Failed to load a `language-legesher-python` package grammar

>filename: `python-traceback.cson`

[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.37.0 x64
**Electron**: 2.0.18
**OS**: Mac OS X 10.14.4
**Thrown From**: [language-legesher-python](https://github.com/legesher/language-legesher-python) package 0.53.4

### Stack Trace

Failed to load a language-legesher-python package grammar

```
At Cannot read property 'type' of null in /Users/madisonedgar/.atom/packages/language-legesher-python/grammars/python-traceback.cson

TypeError: Cannot read property 'type' of null
    at GrammarRegistry.createGrammar (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:360355)
    at l.readFile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:360208)
    at /Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:581297
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:511:3)
  at /packages/language-legesher-python/grammars/python-traceback.cson:1:1
```

### Commands

```
     -0:28.4.0 find-and-replace:show (input.hidden-input)
     -0:24 core:backspace (input.hidden-input)
  3x -0:23.6.0 core:confirm (input.hidden-input)
     -0:20.9.0 project-find:show (input.hidden-input)
     -0:20.6.0 core:confirm (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.33.4 
autocomplete-emojis 2.5.0 
language-legesher-python 0.53.4 
```

**Comment by madiedgar** [comment]
This issue is addressed in Pull Request #6 Grammar File Error

**Comment by madiedgar** [comment]
Not able to reproduce error // will close for now, still undergoing testing for newest update
